frontend/src/metabase/visualizations/click-actions/drills/mlv2/SummarizeColumnDrill.ts

```diff
@@ -0,0 +1,48 @@
+import { t } from "ttag";
+import type {
+  ClickActionBase,
+  Drill,
+} from "metabase/visualizations/types/click-actions";
+import type { Dispatch } from "metabase-types/store";
+import type * as Lib from "metabase-lib";
+
+type AggregationOperator = "sum" | "avg" | "distinct";
```

Comment:
Maybe modify `SummarizeColumnDrillThruInfo` etc and extract types for all enum values, so they can be reused here?